pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
+				return "metadata.name", value, nil
+			case "metadata.name", "metadata.namespace", "status.phase":
```

Comment:
If `"metadata.namespace"` isn't a field of `Namespace`, why is it returned by [NamespaceToSelectableFields()](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/namespace/strategy.go#L146) as a selectable field?

I'll remove it, but NamespaceToSelectableFields() should be fixed.